pkg/handlers/webhookreceiver.go

```diff
@@ -146,17 +145,34 @@ func (h *WebhookReceiverHandler) processAlert(alert template.Alert, mnl *oav1alp
 		// This is not an error state
 		return nil
 	}
+
+	var attempts int = 5
+	var sleep time.Duration = 30
```

Comment:
Was Duration unit is nanoseconds? Just wonder if 30 nanoseconds is longer enough for retry

Reply:
Thanks @tkong-redhat for the review. I have updated this in my latest commit PTAL